contracts/consensus-gateway/ConsensusGateway.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity ^0.5.0;
```

Comment:
Update the compiler version as:
- `pragma solidity >=0.5.0 <0.6.0;`

Reply:
Done 👍